<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPC_View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CameraView" name="graphics_view"/>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>circle_y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_start">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_heading">
          <property name="minimum">
           <double>-999999999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_trailer_angle">
          <property name="minimum">
           <double>-999999999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_circle_y">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>-999999999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>circle_x</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_circle_radius">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>-999999999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>speed:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Trailer angle</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Heading</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>circle_radius:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_speed">
          <property name="minimum">
           <double>-30.000000000000000</double>
          </property>
          <property name="maximum">
           <double>30.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_circle_x">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>-999999999999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraView</class>
   <extends>QGraphicsView</extends>
   <header>mpc_view.camera_view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
